webapp/sass/common.scss

```diff
@@ -0,0 +1,198 @@
+pre,
+code,
+tt {
+  font-family: SFMono-Regular, Consolas, Liberation Mono, Menlo, monospace;
+  font-size: 14px;
+}
+
+tt {
+  border-radius: 3px;
+  border: 1px solidver(--ps-gray-light-4);
+  padding: 2px 4px;
+  background-color: var(--ps-c-white-dark-2);
+}
+
+.navbar {
+  display: flex;
+  height: 60px;
+  align-items: center;
+}
+
+.navbar {
+  .logo {
+    margin: 0;
+    padding: 0;
+    font-weight: 400;
+    background-size: 40px 40px;
+    background-position: 0px 8px;
+    background-repeat: no-repeat;
+  }
+
+  .logo::after {
+    content: 'Pyroscope';
+    font-weight: 400;
+    padding-left: 44px;
+    font-size: 25px;
+    vertical-align: text-top;
+  }
+}
+
+select {
+  @include outline;
+  display: inline-block;
+  position: relative;
+  border-radius: 4px;
+  background-color: var(--ps-c-gray-light-2);
+  border: 1px solid var(--ps-c-gray-light-3);
+
+  &:not([multiple]):not([size]) {
+    background-image: url("data:image/svg+xml;charset=utf-8,%3Csvg xmlns='http://www.w3.org/2000/svg' width='16' fill='white' height='4'%3E%3Cpath d='M4 0h6L7 4'/%3E%3C/svg%3E");
+  }
+}
+
+input {
+  @include outline;
+  display: inline-block;
+  position: relative;
+  border-radius: 4px;
+  background-color: var(--ps-c-gray-light-2);
+  border: 1px solid var(--ps-c-gray-light-3);
+}
+
+.flamegraph-table {
```

Comment:
Where is this one used other than here? Can we scope that to the `FlamegraphTable` component using css modules?